js/build_map.js

```diff
@@ -137,12 +139,11 @@ var categoryButtons = d3.select('#button-container')
   .on('click', function(d){
     prevCategory = activeCategory;
     activeCategory = d.toLowerCase(); // put this here so it only changes on click
-    updateCategory(activeCategory, action = 'click');
+    updateCategory(activeCategory, prevCategory.toLowerCase());
```

Comment:
do we need to call toLowerCase on prevCategory? i think we're doing a pretty good job of ensuring that prevCategory is always lowercase to begin with, since it either comes from activeCategory or is set to 'total'